numpy/linalg/linalg.py

```diff
@@ -1810,7 +1810,7 @@ def det(a):
 
 # Linear Least Squares
 
-def lstsq(a, b, rcond=-1):
+def lstsq(a, b, rcond="warn"):
```

Comment:
It was invented mostly for "keepdims", so I don't know that we really have a tradition yet. I don't feel strongly either way about using it in this case. I thought "warn" was a bit unusual, but not particularly problematic. I confess to a certain weakness for things that are just a bit different, and this was one of them.